src/views/Pools/components/LockedPool/Modals/ExtendDurationModal.tsx

```diff
@@ -40,10 +48,18 @@ const ExtendDurationModal: React.FC<ExtendDurationModal> = ({
         isOverMax: totalDuration > MAX_LOCK_DURATION,
       }
     },
-    [currentLockedAmount, currentDuration],
+    [currentLockedAmount, lockEndTime],
   )
 
-  const prepConfirmArg = useCallback(({ duration }) => ({ finalDuration: duration, finalLockedAmount: 0 }), [])
+  const prepConfirmArg = useCallback(
+    ({ duration }) => ({
+      finalDuration: duration,
+      finalLockedAmount: checkEnoughBalanceToExtend
```

Comment:
My concern is that we quietly block users' CAKE. Even though the amount is small, I think users should be aware of this before taking the action. At the end, it's still their CAKE, their decision.